tus_storagehandler/exceptions.py

```diff
@@ -0,0 +1,22 @@
+"""Tus Storage Handler exceptions."""
+
+from werkzeug.exceptions import BadRequest, InternalServerError, NotFound
+
+exceptions = {
+    Exception: {
+        "message": "An unexpected error occurred. Please try again.",
+        "code": 500,
```

Comment:
It's a nitpick but since you've already replaced all the status code with codes from http module why don't you do it here as well. It's up to you tbh :).

Reply:
I'm actually not entirely sure that would work (FOCA may complain). But give it a try if you like.